build.sh

```diff
@@ -1,49 +1,53 @@
 #!/bin/bash -e
+
+export GRADLE_ARCHIVE=gradle-6.8.1-bin.zip
+export CORRETTO_ARCHIVE=amazon-corretto-15-x64-linux-jdk.tar.gz
+export GHIDRA_VER=9.2.2_PUBLIC
+export GHIDRA_DATE=20201229
+export GHIDRA_ARCHIVE=ghidra_${GHIDRA_VER}_${GHIDRA_DATE}.zip
+export GHIDRA_URL=https://ghidra-sre.org/${GHIDRA_ARCHIVE}
+
 pushd /tmp
 
 echo "[*] Downloading files..."
-cat <<EOF > urls.txt
-https://corretto.aws/downloads/latest/amazon-corretto-11-x64-linux-jdk.tar.gz
-https://services.gradle.org/distributions/gradle-5.0-bin.zip
-https://ghidra-sre.org/ghidra_9.2.1_PUBLIC_20201215.zip
-https://github.com/Cxbx-Reloaded/XbSymbolDatabase/releases/latest/download/XbSymbolDatabase.zip
-EOF
-cat urls.txt | xargs -n 1 -P 10 wget --no-verbose
+wget -nv https://corretto.aws/downloads/latest/${CORRETTO_ARCHIVE}	\
```

Comment:
* This wonky looking `xargs` structure I had is here to let these packages download in parallel. Let's prefer to keep it
* Let's be consistent with the URL-in-environment-variable method as you've done for `GHIDRA_URL`

Reply:
Is there any particular reason it's 10 processes?

Reply:
No particular reason. Just need a number >= number of URLs

Reply:
I changed it to the size of the array.